yt_dlp/extractor/youtube.py

```diff
@@ -17,7 +17,6 @@
 import time
 import traceback
 import urllib.parse
```

Comment:
revert unnecessary change
```suggestion
import urllib.parse

```